docs/Tutorials/CCE.md

```diff
@@ -0,0 +1,107 @@
+\cond NEVER
+Distributed under the MIT License.
+See LICENSE.txt for details.
+\endcond
+# %Running CCE {#tutorial_cce}
+
+The basic instructions for getting up and running with a stand-alone
+CCE using external data are:
+- Clone spectre and build the CharacteristicExtract target
+- *Note*: CCE currently requires the linked HDF5 library to be built
+  thread-safe. As this is a significant restriction on many systems,
+  we hope to eliminate this requirement in the future.
+- At this point (provided the build succeeds) you should have the
+  executable `build_directory/bin/CharacteristicExtract`; You can now
+  run that using an input file that should be patterned on
+  `tests/InputFiles/Cce/CharacteristicExtract.yaml` from the spectre
+  source tree. There are a few important notes there:
+  - for resolution, the example input file has lmax (option group `CCE`,
+    option `LMax`) of 12, and filter lmax (option group `Filtering`, option
+    `FilterLMax`) of 10; that'll run pretty fast but might be a little low for
+    full precision. lmax 16, filter 14 should be pretty good, and typically
+    precision doesn't improve above lmax 24, filter 22 (be sure to
+    update the filter as you update lmax).
+  - if you want to just run through the end of the provided worldtube data,
+    you can just omit the `EndTime` option and the executable will figure it
+    out from the worldtube file.
+  - the `ScriOutputDensity` adds extra interpolation points to the output,
+    which is useful for finite-difference derivatives on the output data, but
+    otherwise it'll just unnecessarily inflate the output files, so if you
+    don't need the extra points, best just set it to 1.
+  - if you're extracting at 100M or less, best to reduce the `TargetStepSize`,
+    to around .5 at 100M and lower yet for nearer extraction.
+  - the `InitializeJ` in the example file uses `InverseCubic` which is a pretty
+    primitive scheme, but early tests indicate that it gives the best results
+    for most systems.
+    If initial data is a concern, you can also try replacing `InverseCubic`
+    entry with :
+    ```
+    NoIncomingRadiation:
+      AngularCoordTolerance: 1.0e-13
+      MaxIterations: 500
+      RequireConvergence: true
+    ```
+    which are probably pretty good choices for those parameters,
+    and the `RequireConvergence: true` will cause the iterative solve in
+    this version to error out if it doesn't find a good frame.
+  .
+- An example of an appropriate submission command for slurm systems is:
+  ```
+  srun -n 1 -c 1 path/to/build/bin/BondiCharacteristicExtract ++ppn 3 \
+ --input-file path/to/input.yaml
+  ```
+  CCE doesn't currently scale to more than 4 cores, so those slurm options are
+  best.
+- CCE will work faster if the input file is chunked in small numbers of
+  complete rows in the hdf5.
```

Comment:
"input file" is already the YAML file, how about "boundary data H5 file"? Then delete `in the hdf5`